### DISCUSSION TITLE
Updating new Tracks

### DISCUSSION THREAD

**Discussion by krish-1010** [the-maazu/react-material-music-player#34] [Q&A]
Greetings, is there a efficient way to change the tracks, from the documentation I understand that Tracks should be hard-coded, is it so? if so is there any ways to update the tracks efficiently using states? Thanks.

**Comment by the-maazu** [comment]
You can do this with beta version. install with react-material-music-player@beta. [additional interface](https://github.com/the-maazu/react-material-music-player/blob/beta/DOCUMENTATION.md)

Edit: You can call the interface and replace tracks multiple times. The beta version gives more control in Javascript.
